packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.10.0
+
+* Bumps webkit from 1.0.0 to 1.5.0.
+* Raises minimum compileSdkVersion to 32. This requires setting the `compileSdkVersion` in the app
```

Comment:
Nit: missing \`s on the first `compileSdkVersion.

Do we need the second sentence? IIRC the `flutter` tool will give a very clear error message about this.